### ISSUE TITLE
http error 500 when implementing version-check workaround

### ISSUE THREAD

**Issue by geotheory** [owncloud/core#24325]
I'm trying to set up Postgres 9.4 with Owncloud 8.1.6.  My previous effort at this last year ran into the version-check problem [(this issue)](https://github.com/owncloud/core/issues/19391) to which someone posted a [workaround](https://github.com/owncloud/core/pull/20524) to implement [this commit](https://github.com/owncloud/core/commit/2f89eef334bd445a7e046d845d5d5d1b3e4b6b8c).

So I've pasted the updated script (with `$logger` lines) to `/var/www/owncloud/lib/private/util.php`, and now when I call the page in e.g. Chrome I get error:
```
The 192.168.1.xxx page isn’t working
192.168.1.xxx is currently unable to handle this request.
HTTP ERROR 500
```
Any idea why?

---------------------------------------------
Not sure if relevant or a separate issue, but when I revert to the original `util.php` and call up the page I can't escape from the error message 
```
Error occurred while checking PostgreSQL version
Please make sure you have PostgreSQL >= 9 or check the logs for more information about the error
```
even if I start from incognito window.  Which means I can't even get back to the option to choose MySQL instead..

**Comment by RealRancor** [comment]
@geotheory Instead of trying to modify the source code on your own why don't you install oC 9.0.1 already including this fix?

For further support questions its probably better to also ask at the forums or the users mailinglist: https://owncloud.org/support/

**Comment by geotheory** [comment]
@RealRancor I'm happy to give oC9 a shot, but does this mean it's still not possible to get Postgres to work with oC8 - which is still the apt-get default for Raspbian Jessie?

**Comment by RealRancor** [comment]
> but does this mean it's still not possible to get Postgres to work with oC8

That fix wasn't backported to oC 8.0/8.1/8.2 so yes, if that fix is needed you can't get these versions to work.

**Comment by Xenopathic** [comment]
Not a bug -> close
